pkg/provision/local_store.go

```diff
@@ -262,6 +286,41 @@ func (s *FSStore) Get(id string) (*Reservation, error) {
 	return s.get(id)
 }
 
+// GetType retrieves a specific reservation's type using its ID
+// if returns a non nil error if the reservation is not present in the store
+func (s *FSStore) GetType(id string) (ReservationType, error) {
```

Comment:
this can be a private function I don't think this method is used anywhere else then in this package.